<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphWindow</class>
 <widget class="QMainWindow" name="GraphWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Monitor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/graph_icon.png</normaloff>:/icons/icons/graph_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="searchText">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::NoInsert</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoFormatButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Auto-Generate Format From Current String</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/icons/cache.png</normaloff>:/icons/icons/cache.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PlotWidget" name="plotWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMonitorFileToggle"/>
   <addaction name="actionReloadFile"/>
   <addaction name="actionClearData"/>
   <addaction name="actionEditConfiguration"/>
  </widget>
  <widget class="QToolBar" name="debugToolBar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTest1"/>
   <addaction name="actionTest2"/>
   <addaction name="actionLoadPresetConfig"/>
  </widget>
  <action name="actionEditConfiguration">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/advanced.png</normaloff>:/icons/icons/advanced.png</iconset>
   </property>
   <property name="text">
    <string>Edit Configuration</string>
   </property>
   <property name="toolTip">
    <string>Edit the graph's configuration</string>
   </property>
  </action>
  <action name="actionTest1">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/agt_update_critical.png</normaloff>:/icons/icons/agt_update_critical.png</iconset>
   </property>
   <property name="text">
    <string>Test 1</string>
   </property>
   <property name="toolTip">
    <string>Test 1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionTest2">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/agt_update_critical.png</normaloff>:/icons/icons/agt_update_critical.png</iconset>
   </property>
   <property name="text">
    <string>Test 2</string>
   </property>
   <property name="toolTip">
    <string>Test 2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionMonitorFileToggle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/player_play.png</normaloff>
     <normalon>:/icons/icons/player_pause.png</normalon>:/icons/icons/player_play.png</iconset>
   </property>
   <property name="text">
    <string>Monitor File</string>
   </property>
   <property name="toolTip">
    <string>Toggle file monitoring</string>
   </property>
  </action>
  <action name="actionLoadPresetConfig">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/openConfig.png</normaloff>:/icons/icons/openConfig.png</iconset>
   </property>
   <property name="text">
    <string>Load Preset Config</string>
   </property>
   <property name="toolTip">
    <string>Load Preset Config</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionReloadFile">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/quick_restart.png</normaloff>:/icons/icons/quick_restart.png</iconset>
   </property>
   <property name="text">
    <string>Reload File</string>
   </property>
   <property name="toolTip">
    <string>Reparse Entire File</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionClearData">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/icons/editdelete.png</normaloff>:/icons/icons/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>Clear Data</string>
   </property>
   <property name="toolTip">
    <string>Clear Data Buffer</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
